graphql/schemabuilder/pagination.go

```diff
@@ -397,31 +398,47 @@ func (c *connectionContext) applyTextFilter(ctx context.Context, nodes []interfa
 		return nodes, nil
 	}
 
-	var filteredNodes []interface{}
-	for _, node := range nodes {
-		keep := false
-		// For each possible field we're matching against, check if there's a match.
-		for name, filterField := range c.FilterTextFields {
-			// Resolve the graphql.Field made for sorting.
-			text, err := filterField.Resolve(ctx, node, nil, nil)
-			if err != nil {
-				return nil, err
-			}
+	nodesToKeep := make([]bool, len(nodes))
 
-			// Only strings are allowed for FilterText fields.
-			textString, ok := text.(string)
-			if !ok {
-				return nil, fmt.Errorf("filter %s returned %T, must be a string", name, text)
-			}
+	g, _ := errgroup.WithContext(ctx)
```

Comment:
why don't we propagate the `ctx` returned?

Reply:
Done